### ISSUE TITLE
Support decoding of error responses without category

### ISSUE THREAD

**Issue by tomasulo** [Sixt/ja-micro#15]
Ja-micro throws a NPE during `RpcCallException.fromJson(error)` if it tries to decode an error response from another service that does not conform with the new error handling and does not contain a category.

Reproducable with this test:

```
@Test
public void testFromJson_OldErrorHandling() {
    String error = "{\"id\":\"com.sixt.service.foobar\",\"code\":500,\"detail\":\"error message\",\"status\":\"Internal Server Error\"}";
    RpcCallException ex = RpcCallException.fromJson(error);
    assertThat(ex).isNotNull();
}
```

**Comment by bkrahmer** [comment]
Hey Thomas, I think we should default the category to InternalServerError in this case.  Do you agree?

**Comment by tomasulo** [comment]
Yes, good idea. I will implement a fix.

**Comment by tomasulo** [comment]
https://github.com/Sixt/ja-micro/pull/17

**Comment by bkrahmer** [comment]
Fixed in 3.0.21
